src/SalmonDE/StatsPE/FloatingTexts/Events/FloatingTextEvent.php

```diff
@@ -26,7 +26,7 @@ public function getFloatingText(): FloatingText{
         return $this->floatingText;
     }
 
-    public function getType(): int{
+    public function getType(){
         return $this->type;
     }
```

Comment:
Should `: int` have been kept? I just thought, `$this->type` was already `int`, so putting `: int` would mean you are putting `int` twice..? I don't know.

Reply:
: int ensures that $this->type can only return int. It should stay there.